docs/04.guides/13.Various/15.FAQs/01.technical-FAQs/02.database-session/page.md

```diff
@@ -44,6 +44,8 @@ That's it!
 
 * Primary key existence (DESCRIBE cf_session_data). If primary key doesn't exist add composite primary key (cfid, name). You can run ALTER TABLE cf_session_data ADD PRIMARY KEY(cfid,name) for that. This helps you avoid performance problems as your cf_session_data table will grow.
 
+* index present on expires column. You'll need it if you have a large trafic bacase purging will check expired records and index will help to do it faster.
```

Comment:
```suggestion
* index present on expires column. You'll need it if you have a large amount of traffic because purging will check expired records and the index will help to do it faster.
```